tests/testcases/015_check-pods-running.yml

```diff
@@ -0,0 +1,33 @@
+---
+- hosts: kube-master[0]
+  tasks:
+
+  - name: Force binaries directory for Container Linux by CoreOS
+    set_fact:
+      bin_dir: "/opt/bin"
+    when: ansible_os_family in ["CoreOS", "Container Linux by CoreOS"]
+
+  - set_fact:
+      bin_dir: "/usr/local/bin"
+    when: not ansible_os_family in ["CoreOS", "Container Linux by CoreOS"]
+
+  - name: Check kubectl output
+    shell: "{{bin_dir}}/kubectl get pods --all-namespaces -owide"
+    register: get_pods
+    no_log: true
+
+  - debug: msg="{{get_pods.stdout.split('\n')}}"
+
+  - name: Pods are running
```

Comment:
You need to allow this task to fail.

Consider to add:

failed_when: false
and
no_log: true

And add a debug task after with:

- debug: msg="{{run_pods_log.stdout.split('\n')}}"